var/spack/repos/builtin/packages/python/package.py

```diff
@@ -272,7 +272,7 @@ def determine_version(cls, exe):
         # Output looks like:
         #   Python 3.7.7
```

Comment:
Can you add a comment here with the output on your system?

Reply:
Pre-patch this is the spack output, will add python in a sec, have to start the image back up:
```
root@ef9d30ceec49:/curdir# spack external find python
==> Warning: error detecting "python" from prefix /bin [Bad characters in version string: 3.10.2+]
==> Warning: error detecting "python" from prefix /usr/bin [Bad characters in version string: 3.10.2+]
==> No new external packages detected
```

Reply:
Here's the python output:
```
root@0593d449d500:/curdir# python3 -V
Python 3.10.2+
```